bazel/BUILD

```diff
@@ -1,3 +1,8 @@
 licenses(["notice"])  # Apache 2
 
 package(default_visibility = ["//visibility:public"])
+
+config_setting(
+    name = "zipkin_disabled",
+    values = {"define": "zipkin_disabled=1"},
```

Comment:
Nit: it would be clearer in the code to make this `ZIPKIN_ENABLED` and invert the logic.

Reply:
I inverted this with respect to the define we add later on and consume in the c++ code. So that part is better.. But at this place I left it as-is pending a question I left on Slack; I'm a bit hesitant to make this opt-in instead of opt-out.